docs/scRNAseq_tutorials.md

```diff
@@ -0,0 +1,25 @@
+## Become more familiar with scRNAseq analysis thanks to these tutorials
+
+Here are some recommandations on learning how to perform scRNAseq analysis. You can find some tutorials on Galaxy or on R.
+
+### Galaxy Environment
+If you want to use the Galaxy environment but you don't know how to use it, we recommend [the following tutorials](https://training.galaxyproject.org/training-material/topics/introduction/)
+
+- *Introduction to Galaxy*
+- *A short introduction to Galaxy*
+- *Options for using Galaxy*
+
```

Comment:
Put the urls. This is the added value of the manuel: save some google searches for readers...

Reply:
Pour les galaxy trainings il y a plusieurs liens à mettre car pour chaque topic il y a plusieurs ressources (avec des liens différents du coup) en cliquant sur le lien précédant les bullet points on tombe sur la table avec les noms des trainings et les différentes ressources.